tests-examples/demo-todo-app.spec.js

```diff
@@ -0,0 +1,449 @@
+// @ts-check
```

Comment:
Not sure if we need examples here :) Its not crucial, but you have raised the bar with this PR, so lets keep it perfect :)

Reply:
Please delete excessive files and commit+push the deletion.